namespace/parser.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2020 Sevki <s@sevki.org>. All rights reserved.
```

Comment:
Best to replace with a non-personal license :)

Reply:
![image](https://media.giphy.com/media/xT5LMzIK1AdZJ4cYW4/giphy.gif)